### DISCUSSION TITLE
Chaos Exploit

### DISCUSSION THREAD

**Discussion by 3kh0** [3kh0/ext-remover#356] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
This was made by xlak, i own nothing!
If you have any issues contact Xlak on https://reddit.com/u/xlak_ or https://github.com/xlak

Steps
1 Go to https://xlak.github.io/chaos

2 Press the begin button and paste the script in the url bar.

3 Go to inspect and click on console. Then paste the script

4 A page should come up saying exploit in progress. Then another one should come up and an inspect console tab should open.

5 Paste the script again and hit enter. You should now see a page saying " Exploit Success " or something. Now check your extension page to see if any of the extensions are disabled.

To turn them back on just paste chrome://restart in the url bar.

EXTRA NOTES
Xlak has a alot of bypasses and proxies once you have disabled your spyware, check them out here ! https://xlak.github.io/hapara-delete

`
Fetched from https://discord.com/channels/419123358698045453/1059602707679297596
`

**Comment by Aka-but-nice** [comment]
I wish I had inspect, but its 2023
